### PR TITLE
changed namespace for ns

### PR DESCRIPTION
**Pull request by Mixaster995**
Signed-off-by: Mikhail Avramenko <avramenkomihail15@gmail.com>

## Issue 
https://github.com/networkservicemesh/sdk-k8s/issues/252

### CODE REVIEW

File: examples/features/nse-composition/nse-composition-ns.yaml [hunk 1]
```diff
@@ -3,7 +3,6 @@ apiVersion: networkservicemesh.io/v1
 kind: NetworkService
 metadata:
   name: nse-composition
-  namespace: nsm-system
```
[the remark this answers is not in the document]
Comment (denis-tingaikin) [reply examples/features/nse-composition/nse-composition-ns.yaml:6]: I looked at it so quickly. OK, now I see.

**Comment by denis-tingaikin** [comment]
Please resolve conflicts.

**Comment by Mixaster995** [comment]
@denis-tingaikin  ok, done

**Comment by Mixaster995** [comment]
@denis-tingaikin resolved conflicts

**Review by denis-tingaikin — APPROVED** [review]

**Pull request merged**